ts/dx.all.d.ts

```diff
@@ -5901,6 +5895,17 @@ declare module DevExpress.viz {
         /** @name BaseLegend.visible */
         visible?: boolean;
     }
+    /** @name BaseLegendItem */
+    export interface BaseLegendItem {
```

Comment:
Just to make sure: should all the derived types (`BarGaugeLegendItem`,`BaseChartLegendItem`, `FunnelLegendItem `, etc.) have all these options (`markerAttributes`, `size`, `text`, `visible`)?

Reply:
Yes, these properties should be in all LegendItems

Reply:
👌